apps/cvicko-map/src/components/App.tsx

```diff
@@ -61,6 +61,17 @@ export const App = () => {
   const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null);
   const [isMobile, setMobile] = useState<boolean | null>(null);
 
+  // set event listeners
+  useEffect(() => {
+    const cancelAnimation = (event: KeyboardEvent) => {
+      if (event.keyCode === 27) stopAnimation();
+    };
+
+    document.addEventListener('keydown', cancelAnimation);
+
+    return () => document.removeEventListener('keydown', cancelAnimation);
+  }, []);
```

Comment:
`stopAnimation` should be presented in the depencencies array. Also your IDE should warn you about this according to our eslint setup where this rule is set. Please check your eslint setup.